gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -279,56 +282,99 @@ MM_VerboseHandlerOutput::handleInitialized(J9HookInterface** hook, uintptr_t eve
+	buffer->formatAndOutput(env, 1, "<attribute name=\"numaNodes\" value=\"%zu\" />", _extensions->_numaManager.getAffinityLeaderCount());
+
+	outputInitializedInnerStanza(env, buffer);
+
+	buffer->formatAndOutput(env, 1, "<system>");
+	buffer->formatAndOutput(env, 2, "<attribute name=\"physicalMemory\" value=\"%llu\" />", omrsysinfo_get_physical_memory());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"numCPUs\" value=\"%zu\" />", omrsysinfo_get_number_CPUs_by_type(OMRPORT_CPU_ONLINE));
+	buffer->formatAndOutput(env, 2, "<attribute name=\"architecture\" value=\"%s\" />", omrsysinfo_get_CPU_architecture());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"os\" value=\"%s\" />", omrsysinfo_get_OS_type());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"osVersion\" value=\"%s\" />", omrsysinfo_get_OS_version());
+	buffer->formatAndOutput(env, 1, "</system>");
+
+	writeVmArgs(env,buffer);
+
+	buffer->formatAndOutput(env, 0, "</initialized>");
+}
+
+void
+MM_VerboseHandlerOutput::outputInitializedRegion(MM_EnvironmentBase *env, MM_VerboseBuffer *buffer)
+{
+	OMR_VM *omrVM = env->getOmrVM();
+#if defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	bool isArrayletDoubleMapRequested = _extensions->isArrayletDoubleMapRequested;
+	const char *arrayletDoubleMappingStatus = _extensions->indexableObjectModel.isDoubleMappingEnabled() ? "enabled" : "disabled";
+	const char *arrayletDoubleMappingRequested = isArrayletDoubleMapRequested ? "true" : "false";
+#endif /* OMR_GC_DOUBLE_MAP_ARRAYLETS */
+	buffer->formatAndOutput(env, 1, "<region>");
+	buffer->formatAndOutput(env, 2, "<attribute name=\"regionSize\" value=\"%zu\" />", _extensions->getHeap()->getHeapRegionManager()->getRegionSize());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"regionCount\" value=\"%zu\" />", _extensions->getHeap()->getHeapRegionManager()->getTableRegionCount());
+	buffer->formatAndOutput(env, 2, "<attribute name=\"arrayletLeafSize\" value=\"%zu\" />", omrVM->_arrayletLeafSize);
+#if defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	if (_extensions->isVLHGC()) {
+		buffer->formatAndOutput(env, 2, "<attribute name=\"arrayletDoubleMappingRequested\" value=\"%s\"/>", arrayletDoubleMappingRequested);
+		if (isArrayletDoubleMapRequested) {
+			buffer->formatAndOutput(env, 2, "<attribute name=\"arrayletDoubleMapping\" value=\"%s\"/>", arrayletDoubleMappingStatus);
+		}
+	}
+#endif /* OMR_GC_DOUBLE_MAP_ARRAYLETS */
+	buffer->formatAndOutput(env, 1, "</region>");
+}
 
-	writer->formatAndOutput(env, 0, "</initialized>\n");
+void
+MM_VerboseHandlerOutput::handleInitialized(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
+{
+	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
+	MM_VerboseWriterChain* writer = _manager->getWriterChain();
+	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+ 	_manager->setInitializedTime(omrtime_hires_clock());
```

Comment:
`event->timestamp` instead of  `omrtime_hires_clock `

Reply:
and then you can also remove `OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());`